lib/util/mime_utils.js

```diff
@@ -42,9 +43,13 @@ shaka.util.MimeUtils = class {
    */
   static getFullOrConvertedType(mimeType, codecs, contentType) {
     const fullMimeType = shaka.util.MimeUtils.getFullType(mimeType, codecs);
+    const ContentType = shaka.util.ManifestParserUtils.ContentType;
 
     if (!shaka.dependencies.muxjs() ||
         !shaka.media.Transmuxer.isTsContainer(fullMimeType)) {
+      if (contentType == ContentType.AUDIO) {
+        return fullMimeType.replace('video', 'audio');
```

Comment:
I went back and forth for a while whether it would make more sense to do this conversion logic before the two times that `getFullOrConvertedType` is called for multiplexed content, instead. E.g. before these two calls:

https://github.com/shaka-project/shaka-player/blob/6084ca6395fbe3d5a97fa92137b8bb51f15c89f8/lib/util/stream_utils.js#L450-L451

https://github.com/shaka-project/shaka-player/blob/6084ca6395fbe3d5a97fa92137b8bb51f15c89f8/lib/util/stream_utils.js#L572-L573

However, on the balance I think this approach makes enough sense, overall. After all, this method is called getFullOr**Converted**Type, so it's not too out there that it might rename the mime type a little.

Instead, I will just ask you to update the JSDoc of this method. Make it clear that it will change video mimeTypes to their audio equivalents if the content type is audio, and specify that is for multiplexed content.